nextjs-logs-manager/app/components/dashboard_elements.tsx

```
@@ -0,0 +1,58 @@
+import { TableRow, TableCell } from "flowbite-react";
+
+interface DashboardElementProps {
+  datetime: Date;
+  type: string;
+  service_name: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)